elements/lisk-p2p/src/p2p_types.ts

```diff
@@ -53,55 +63,29 @@ export interface P2PInternalState {
 	readonly productivity?: number;
 	readonly reputation?: number;
 	readonly connectionKind?: ConnectionKind;
-	readonly advertiseAddress: boolean;
 }
 
-export interface P2PPeerInfo {
+export interface P2PBasicPeerInfo {
```

Comment:
We are falling back to old p2p types that we removed i.e., having two 2 peer types for holding peer information

Reply:
I've added `P2PBasicPeerInfo` because it is handy inside the `PeerBook` to get and remove peers by peer id and ip. We don't have peer info splitted into different places. `P2PPeerInfo` is the only one with a proper structure.